examples/write_sample.rs

```diff
@@ -27,15 +27,15 @@ fn doit(filename: &str) -> zip_next::result::ZipResult<()> {
 
     let mut zip = zip_next::ZipWriter::new(file);
 
-    zip.add_directory("test/", Default::default())?;
+    zip.add_directory("test/", SimpleFileOptions::default())?;
 
-    let options = FileOptions::default()
+    let options = SimpleFileOptions::default()
         .compression_method(zip_next::CompressionMethod::Stored)
         .unix_permissions(0o755);
     zip.start_file("test/☃.txt", options)?;
     zip.write_all(b"Hello, World!\n")?;
 
-    zip.start_file("test/lorem_ipsum.txt", Default::default())?;
+    zip.start_file("test/lorem_ipsum.txt", FileOptions::<()>::default())?;
```

Comment:
Can't the `options` from line 32 be reused here since they're `Copy`?

Reply:
Yeah. I just forgot to fix it.